src/asim/configs/resident/vehicle_type_choice_op4_coefficients.csv

```diff
@@ -192,5 +192,5 @@ coef_calib_pev_2022,-2.55829,F
 coef_calib_bev_2022,-1.70775,F
 coef_calib_pev_2035,-12.8722,F
 coef_calib_bev_2035,-12.75503,F
-coef_calib_pev_2050,1.1943718882487044,F
-coef_calib_bev_2050,0.16922539004489656,F
+coef_calib_pev_2050,-67.86506714368133,F
```

Comment:
Please reduce number of significant digits in future. It helps with readability and usually does not impact the results at all.

Reply:
@JoeJimFlood in future, it might help to include information about changes to the number of chargers since this is such a signification change in the coefficients.